dotcom-rendering/src/components/marketing/epics/ContributionsEpicTicker.tsx

```diff
@@ -0,0 +1,199 @@
+import { palette } from '@guardian/source-foundations';
+import { textSans } from '@guardian/source-foundations';
+import type { TickerSettings } from '@guardian/support-dotcom-components/dist/shared/src/types';
+import { useEffect, useState } from 'react';
+import { useIsInView } from '../../../lib/useIsInView';
+import { useTicker } from '../hooks/useTicker';
+import type { ReactComponent } from '../lib/ReactComponent';
+
+// This ticker component provides an animated progress bar and counter for the
+// epic. It mirrors the behaviour of the "unlimited" ticker type from frontend.
+// The "hardstop" type is not supported. The differences between the two relate
+// to behaviour once the goal has been reached.
+
+const rootStyles = css`
+	position: relative;
+	height: 65px;
+	margin-bottom: 15px;
+	line-height: 18px;
+`;
+
+const totalCountStyles = css`
+	${textSans.medium({ fontWeight: 'bold' })}
+`;
+
+const soFarCountStyles = css`
+	${textSans.medium({ fontWeight: 'bold' })}
+`;
+
+const countLabelStyles = css`
+	${textSans.medium()}
+`;
+
+const progressBarHeight = 10;
+
+const progressBarContainerStyles = css`
+	width: 100%;
+	height: ${progressBarHeight}px;
+	/* stylelint-disable-next-line color-no-hex */
+	background-color: #dda7a1;
+	position: absolute;
+	bottom: 0;
+	margin-top: 40px;
+`;
+
+const progressBarStyles = css`
+	overflow: hidden;
+	width: 100%;
+	height: ${progressBarHeight}px;
+	position: absolute;
+`;
+
+const soFarContainerStyles = css`
+	position: absolute;
+	left: 0;
+	bottom: ${progressBarHeight + 5}px;
+`;
+
+const progressBarTransform = (
+	end: number,
+	runningTotal: number,
+	total: number,
+): string => {
+	const haveStartedAnimating = runningTotal > 0;
+
+	if (!haveStartedAnimating) {
+		return 'translateX(-100%)';
+	}
+
+	const percentage = (total / end) * 100 - 100;
+
+	return `translate3d(${percentage >= 0 ? 0 : percentage}%, 0, 0)`;
+};
+
+const filledProgressStyles = (
+	end: number,
+	runningTotal: number,
+	total: number,
+): SerializedStyles => css`
+	position: absolute;
+	top: 0;
+	left: 0;
+	right: 0;
+	bottom: 0;
+	transform: ${progressBarTransform(end, runningTotal, total)};
+	transition: transform 3s cubic-bezier(0.25, 0.55, 0.2, 0.85);
+	background-color: ${palette.news[400]};
+`;
+
+const goalContainerStyles: SerializedStyles = css`
+	position: absolute;
+	right: 0;
+	bottom: ${progressBarHeight + 5}px;
+	text-align: right;
+`;
+
+const goalMarkerStyles = (transform: string): SerializedStyles => css`
+	border-right: 2px solid ${palette.neutral[7]};
+	content: ' ';
+	display: block;
+	height: 12px;
+	margin-top: -2px;
+	transform: ${transform};
+`;
+
+type MarkerProps = {
+	goal: number;
+	end: number;
+};
+
+const Marker: ReactComponent<MarkerProps> = ({ goal, end }: MarkerProps) => {
+	if (end > goal) {
+		const markerTranslate = (goal / end) * 100 - 100;
+		const markerTransform = `translate3d(${markerTranslate}%, 0, 0)`;
+
+		return <div css={goalMarkerStyles(markerTransform)} />;
```

Comment:
Not a blocker, but as per Emotion best practice, [you should use the style prop for dynamic styles](https://emotion.sh/docs/best-practices#use-the-style-prop-for-dynamic-styles).

Reply:
👍 health card created